image/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
```

Comment:
could we use rhel here instead

Reply:
I think you need RHEL host with subscription to build it. So we would need to build in private jenkins instance.